### ISSUE TITLE
Cypress detected that you returned a promise in a test, but also invoked one or more cy commands inside of that promise.

### ISSUE THREAD

**Issue by kpturner** [TheBrainFamily/cypress-cucumber-typescript-example#9]
Seems to be caused by the inclusion of `common/globalBefore.ts`.
This warning occurs in the browser console when running tests (even if the test actually doesn't do an awful lot).

The issue can be seen in this PR: https://github.com/TheBrainFamily/cypress-cucumber-typescript-example/pull/8

Run

`npm run test:open`

Run the only feature test available.  Look at the browser console output to see numerous examples

```
cypress_runner.js:179344 Cypress Warning: Cypress detected that you returned a promise in a test, but also invoked one or more cy commands inside of that promise.

The test title was:

  > Passing with TypeScript "before each" hook

While this works in practice, it's often indicative of an anti-pattern. You almost never need to return both a promise and also invoke cy commands.

Cy commands themselves are already promise like, and you can likely avoid the use of the separate Promise.

```
If your remove the `common/globalBefore.ts` file the warnings go away.

**Comment by kpturner** [comment]
Looks like a Cypress bug - sorry

https://github.com/cypress-io/cypress/issues/3139
